default.nix

```diff
@@ -96,24 +98,68 @@ let
       ghc = pkgs.haskell.compiler.ghc843;
     };
 
+    playgroundGhc = pkgs.haskell.packages.ghc843.ghcWithPackages (ps: [
+      haskellPackages.plutus-playground-server
+      haskellPackages.plutus-playground-lib
+      haskellPackages.plutus-use-cases
+    ]);
+
     localPackages = localLib.getPackages {
       inherit (self) haskellPackages; filter = localLib.isPlutus;
     };
     tests = {
       shellcheck = pkgs.callPackage localLib.iohkNix.tests.shellcheck { inherit src; };
       hlint = pkgs.callPackage localLib.iohkNix.tests.hlint {
         inherit src;
-        projects = localLib.plutusPkgList;
+        projects = let
+                     fixPlaygroundServer = v: if v != "plutus-playground-server" then v else "plutus-playground/plutus-playground-server";
+                     fixPlaygroundLib = v: if v != "plutus-playground-lib" then v else "plutus-playground/plutus-playground-lib";
+                   in
+                     map (comp fixPlaygroundServer fixPlaygroundLib) localLib.plutusHaskellPkgList;
       };
       stylishHaskell = pkgs.callPackage localLib.iohkNix.tests.stylishHaskell {
         inherit (self.haskellPackages) stylish-haskell;
         inherit src;
       };
     };
+    plutus-server-invoker = pkgs.stdenv.mkDerivation {
```

Comment:
I think ideally some of these would live in other files and be `callPackage`'d in.